src/lib/viewers/doc/__tests__/DocBaseViewer-test.js

```diff
@@ -2946,17 +2946,21 @@ describe('src/lib/viewers/doc/DocBaseViewer', () => {
 
         describe('handleAnnotationColorChange', () => {
             beforeEach(() => {
+                docBase.annotator = {
+                    emit: jest.fn(),
+                };
                 docBase.annotationModule = {
                     setColor: jest.fn(),
                 };
                 docBase.renderUI = jest.fn();
             });
 
-            test('should call setColor and renderUI', () => {
+            test('should call setColor and renderUI, and emit color', () => {
                 const color = '#fff';
                 docBase.handleAnnotationColorChange(color);
 
                 expect(docBase.annotationModule.setColor).toBeCalledWith(color);
+                expect(docBase.annotator.emit).toBeCalledWith('annotations_stroke_set', { color: '#fff' });
```

Comment:
Can we reuse `ANNOTATOR_EVENT.setStroke`?

Reply:
Maybe we can just pass in color instead of redefining it on L726? It's defined on L722.